docs/cow-protocol/reference/core/auctions/competition_rules.md

```diff
@@ -43,7 +43,7 @@ At CoW DAO's discretion, systematic violation of these rules may lead to penaliz
   - buy token / base token
   - sell token / buy token
 
-  The following detail sections list the protocols and base tokens that are considered for Ethereum Mainnet and Gnosis Chain:
+  <a name="base_tokens"></a>The following detail sections list the protocols and base tokens that are considered for Ethereum Mainnet and Gnosis Chain:
```

Comment:
I admit i don't really think this works but i don't know how to reference this paragraph from another file. Any help would be appreciated.